### PR TITLE
Add breadcrumbs

### PR DESCRIPTION
**Pull request by richtier**
image

### CODE REVIEW

File: demo/templates/demo/breadcrumbs.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        <h2 class="heading-medium">Three levels</h2>
+        <pre><code>{% verbatim %}{% breadcrumbs left_url='http://www.example.com' left_label='home' middle_url='http://www.example.com/middle/' middle_label='middle label' right_label='right label' %}{% endverbatim %}</code></pre>
+        {% breadcrumbs left_url='http://www.example.com' left_label='home' middle_url='http://www.example.com/middle/' middle_label='middle label' right_label='right label' %}
+      </div>
+    </div>
+    <div class="grid-row"> 
+      <div class="column-full padding-bottom-15">
+        <h2 class="heading-medium">Two levels</h2>
+        <p>Similar to three levels, but exclude middle_url and middle_label.</p>
+        <pre><code>{% verbatim %}{% breadcrumbs left_url='http://www.example.com' left_label='home' middle_url='http://www.example.com/middle/' middle_label='middle label' right_label='right label' %}{% endverbatim %}</code></pre>
```
Comment (MichaelWalker) [inline demo/templates/demo/breadcrumbs.html]: This still contains the 'middle_label' and 'middle_url' attributes.
Should they be excluded from this fragment?

File: directory_components/templates/directory_components/breadcrumbs_flat.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<div class="breadcrumbs">
+  <ol>
+    <li>
+      <a href="{{ left_url }}">{{ left_label }}</a>
+    </li>
+    {% if middle_label %}
+	    <li>
+	      <a href="{{ middle_url }}">{{ middle_label }}</a>
+	    </li>
+	{% endif %}
```
Comment (MichaelWalker) [inline directory_components/templates/directory_components/breadcrumbs_flat.html]: slightly odd indentation here if we are being super picky! 😊

File: directory_components/templates/directory_components/breadcrumbs_flat.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<div class="breadcrumbs">
```
Comment (MichaelWalker) [inline directory_components/templates/directory_components/breadcrumbs_flat.html]: Do we ever have more that 3 levels to handle? 
I can't think of an example of where we do, but don't know the site well enough to be sure yet!

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+
+## 10.6.0
```
Comment (MichaelWalker) [inline CHANGELOG.md]: Possibly missing something obvious here, but I don't see where the version has been updated.

Comment (MichaelWalker) [reply CHANGELOG.md]: Also this is now slightly out of date - current version of directory components is now 10.7.0

File: demo/templates/demo/breadcrumbs.html (hunk 1 above)
Comment (richtier) [reply demo/templates/demo/breadcrumbs.html]: thanks. yep :)

File: directory_components/templates/directory_components/breadcrumbs_flat.html (hunk 3 above)
Comment (richtier) [reply directory_components/templates/directory_components/breadcrumbs_flat.html]: tl;dr you're correct. I'll change stuff.

we used to use `breadcrumbs.html`, which supports multi levels. I think it was used by CMS pages. Looks like it's not used anwyere now https://github.com/search?l=HTML&q=org%3Auktrade+directory_components%2Fbreadcrumbs.html&type=Code

will simplify this stuff now

File: CHANGELOG.md (hunk 4 above)
Comment (richtier) [reply CHANGELOG.md]: I think I made this PR before another PR was made and merged which also used 10.6.0...maybe

will use 10.8.0 :)

**Pull request merged**